src/frontend/i18n/locales/fr-FR.json

```diff
@@ -779,6 +775,146 @@
+    "message": "All my courses"
+  },
+  "components.TeacherDashboard.TeacherDashboardRoutes.profile.courses.path": {
+    "description": "The path to display the teacher courses liste view.",
+    "message": "/teacher/profile/courses"
+  },
+  "components.TeacherDashboard.TeacherDashboardRoutes.profile.label": {
+    "description": "Label of the teacher profile view.",
+    "message": "Profile"
+  },
+  "components.TeacherDashboard.TeacherDashboardRoutes.profile.notifications.label": {
+    "description": "Label of the teacher notifications view.",
+    "message": "Notifications"
+  },
+  "components.TeacherDashboard.TeacherDashboardRoutes.profile.notifications.path": {
+    "description": "The path to display the teacher notifications view.",
+    "message": "/teacher/profile/notifications"
+  },
+  "components.TeacherDashboard.TeacherDashboardRoutes.profile.path": {
+    "description": "The path to display the teacher profile view.",
+    "message": "/teacher/profile"
+  },
+  "components.TeacherDashboard.TeacherDashboardRoutes.profile.settings.label": {
+    "description": "Label of the teacher profile settings view.",
+    "message": "Settings"
+  },
+  "components.TeacherDashboard.TeacherDashboardRoutes.profile.settings.path": {
+    "description": "The path to display the teacher profile settings view.",
+    "message": "/teacher/profile/settings"
+  },
+  "components.TeacherDashboard.TeacherDashboardRoutes.root.label": {
+    "description": "Label of the teacher dashboard root view.",
+    "message": "Teacher dashboard"
+  },
+  "components.TeacherDashboard.TeacherDashboardRoutes.root.path": {
+    "description": "The path to display the teacher dashboard root view.",
+    "message": "/teacher"
+  },
+  "components.TeacherOrganizationCourseDashboardLoader.loading": {
+    "description": "Message displayed while loading an organization",
+    "message": "Loading organization ..."
+  },
+  "components.TeacherOrganizationDashboardSidebar.header": {
+    "description": "Title of the organization dashboard sidebar",
+    "message": "{organizationName}"
+  },
+  "components.TeacherOrganizationDashboardSidebar.subHeader": {
+    "description": "Sub title of the organization dashboard sidebar",
+    "message": "You are on the organization dashboard"
+  },
+  "components.TeacherProfileDashboardLoader.loading": {
+    "description": "Message displayed while loading profile",
+    "message": "Loading profile ..."
+  },
+  "components.TeacherProfileDashboardSidebar.header": {
+    "description": "Title of the dashboard sidebar",
+    "message": "{name}"
+  },
+  "components.TeacherProfileDashboardSidebar.subHeader": {
+    "description": "Sub title of the dashboard sidebar",
+    "message": "You are on your teacher dashboard"
+  },
+  "components.TeacherProfileSettingsDashboardLoader.loading": {
+    "description": "Message displayed while loading settings",
+    "message": "Loading settings..."
+  },
```

Comment:
you're aware that all this is not translated?

Reply:
Yup but this feature is currently in development and translation can change so it is not mandatory to have it right now.